app/views/admin/merchandise/variants/show.html.erb

```diff
@@ -35,7 +35,7 @@
   <%= label :variant, :count_on_hand %>
 </div>
 <div>
-  <%= @variant.count_on_hand %>
+  <%= @variant.inventory.count_on_hand %>
```

Comment:
this method already points to the inventory object

https://github.com/drhenner/ror_ecommerce/blob/b7142f06275002be15ee2257862c2e7fd3a6a8b2/app/models/variant.rb#L53C1-L67C60